Backend/User/src/main/java/user/service/ProfileService.java

```diff
@@ -0,0 +1,240 @@
+package user.service;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import user.dto.ProfileRequest;
+import user.model.User;
+import user.repository.JWTokenRepository;
+import user.repository.UserRepository;
+import user.util.JwtUtil;
+import user.util.UserNotFoundException;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+@Service
+public class ProfileService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private JWTokenRepository jwTokenRepository;
+
+    @Autowired
+    private JwtUtil jwtUtil;
+
+    @Autowired
+    private BCryptPasswordEncoder passwordEncoder;
+
+    @Transactional
+    public boolean createProfile(ProfileRequest profileRequest) {
+        String email = profileRequest.getEmail();
+        String password = profileRequest.getPassword();
+        String name = profileRequest.getName();
+        Byte isAdmin = profileRequest.getIsAdmin();
+
+        // Validate email and password
+        if (email == null || password == null) {
+            throw new IllegalArgumentException("Email and password are required");
+        }
+
+        if (name == null) {
+            throw new IllegalArgumentException("Name is required");
+        }
+
+        if (isAdmin == null) {
+            throw new IllegalArgumentException("Role is required");
+        }
+
+        // Validate isAdmin
+        if (isAdmin != 0 && isAdmin != 1) {
+            throw new IllegalArgumentException("Invalid value for isAdmin. It must be 0 or 1.");
+        }
+
+        // Check if the email already exists
+        String existingUUID = userRepository.checkEmail(email);
+        if (existingUUID != null) {
```

Comment:
if u wanna u can consider validating with a RegExp pattern to make sure its a valid email expression 
eg. /w@/w ( something like that is a basic one )